src/main/java/org/dependencytrack/event/kafka/processor/VulnerabilityScanResultProcessor.java

```diff
@@ -607,16 +613,37 @@ private List<Vulnerability> maybeApplyPolicyAnalyses(final Dao dao, final Compon
                                 .map(comment -> new AnalysisComment(createdAnalysis.id(), comment.comment(), comment.commenter()))
                                 .toList());
             }
-
             dao.createAnalysisComments(analysisCommentsByVulnId.values().stream().flatMap(Collection::stream).toList());
         }
 
+        // dispatch PROJECT_AUDIT_CHANGE notifications
+        maybeSendProjectAuditChangeNotification(component.projectUuid, projectAuditChangeNotifications);
+
         return vulnById.entrySet().stream()
                 .filter(entry -> newFindingVulnIds.contains(entry.getKey()))
                 .map(Map.Entry::getValue)
                 .toList();
     }
 
+    private org.dependencytrack.proto.notification.v1.Notification createAndAddProjectAuditChangeNotification(QueryManager qm, Component component, Vulnerability vuln, Analysis policyAnalysis, boolean analysisStateChange, boolean suppressionChange) {
+        final Timestamp notificationTimestamp = Timestamps.now();
+        AtomicReference<org.dependencytrack.proto.notification.v1.Notification> notification = new AtomicReference<>();
```

Comment:
Usage of `AtomicReference` can be avoided by using `jdbi(qm).withExtension` instead. That way you can return the query result directly, just like it's done here: 

https://github.com/DependencyTrack/hyades-apiserver/blob/281b3794b4b28d9a5afc85a65bc35b906c2b3d6b/src/main/java/org/dependencytrack/event/kafka/processor/VulnerabilityScanResultProcessor.java#L126